src/store/models/environment.ts

```diff
@@ -89,5 +92,24 @@ export default createModel<RootModel>()({
         sendToBitgo(BitgoAction.TOKEN_V2_INFO_TOKEL, { tokenId })
       );
     },
+    async getContentType({ tokenid, url }) {
+      try {
+        if (IS_IPFS(url)) {
+          return this.SET_TOKEN_CONTENT_TYPE({ tokenid, type: 'ipfs' });
+        }
+        const response = await axios(url);
```

Comment:
If I understand correctly, this will cause us to fetch media twice (once to check headers, a second time to actually display the URL).

This is fine imo, but we may want to force this axios request to be a `HEAD` instead of `GET` so we are not fetching the body unnecessarily. Downside is some servers may not support HEAD at all or respond with different headers than a GET request.

Reply:
good point! I ll change it to HEAD request and if I don't get the header that I need I will send a following GET request. That way we will only rarely double fetch the media.

In the future we can optimize if necessary.

Reply:
sounds like a good approach!